CHANGELOG.md

```diff
@@ -30,6 +30,9 @@ to behave like `helm-find-files`, such as multifile selection and opening or del
   buffers by name
 * New `defcustom` `projectile-globally-ignored-file-suffixes` allows
   you to globally ignore files with particular extensions
+* Use file-truename when caching filenames to prevent
```

Comment:
Yes.

On Saturday, April 25, 2015, IvanMalison <notifications@github.com> wrote:

> In CHANGELOG.md
> <https://github.com/bbatsov/projectile/pull/691#discussion_r29098818>:
>
> > @@ -30,6 +30,9 @@ to behave like `helm-find-files`, such as multifile selection and opening or del
> >    buffers by name
> >  * New `defcustom` `projectile-globally-ignored-file-suffixes` allows
> >    you to globally ignore files with particular extensions
> > +* Use file-truename when caching filenames to prevent
>
> Okay, so really, what you mean is that this needs to be merged with
> master...
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/bbatsov/projectile/pull/691/files#r29098818>.
>


-- 
Best Regards,
Bozhidar Batsov

http://www.batsov.com